### PR TITLE
tests: fix incompatibility with pytest>=2.8.0

### PR DESCRIPTION
**Pull request by SamiHiltunen**
* FIX Removes calls to PluginManager
  consider_setuptools_entrypoints() removed in PyTest 2.8.0.

Signed-off-by: Sami Hiltunen <sami.mikael.hiltunen@cern.ch>

### CODE REVIEW

**Comment by jirikuncar** [comment]
- [ ] update travis with `sudo: false`
- [ ] fix broken imports:
```
==================================== ERRORS ====================================
______________________ ERROR collecting tests/test_api.py ______________________
tests/test_api.py:22: in <module>
    from invenio_ext.restful.utils import APITestCase
../../../virtualenv/python2.7.9/lib/python2.7/site-packages/invenio_ext/restful/utils.py:27: in <module>
    from invenio.testsuite import InvenioTestCase
E   ImportError: No module named invenio.testsuite
___________________ ERROR collecting tests/test_receivers.py ___________________
tests/test_receivers.py:30: in <module>
    from invenio_testing import InvenioTestCase
../../../virtualenv/python2.7.9/lib/python2.7/site-packages/invenio_testing/__init__.py:30: in <module>
    from flask_testing import TestCase
E   ImportError: No module named flask_testing
--------------- coverage: platform linux2, python 2.7.9-final-0 ----------------
```

**Comment by SamiHiltunen** [comment]
@jirikuncar https://github.com/inveniosoftware/invenio-ext/pull/12 probably fixes at least those errors

**Comment by jirikuncar** [comment]
- [ ] `.travis.invenio.cfg` missing (do we need `invenio_search`?)

File: .travis.yml [hunk 1]
```diff
@@ -35,18 +37,16 @@ cache:
...
 
 python:
   - "2.7"
 
 before_install:
-  - "sudo apt-get update"
-  - "sudo apt-get install python-dev"
   - "travis_retry pip install --upgrade pip"
-  - "travis_retry pip install mock"
+  - "travis_retry pip install check-manifest mock"
```
Comment (jirikuncar) [inline .travis.yml]: `coveralls`?

File: .travis.invenio.cfg [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+CFG_BIBSCHED_PROCESS_USER = getpass.getuser()
+
+DEBUG = False
+SECRET_KEY = 'MY_SECRET'
+
+# Disable all automatic asset building - false is /usr/bin/false.
+ASSETS_AUTO_BUILD = False
+
+PACKAGES = [
+    'invenio-oauth2server',
```
Comment (jirikuncar) [inline .travis.invenio.cfg]: typo?

File: .travis.invenio.cfg [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    'invenio_records',
+    'invenio_testing',
+    'invenio_celery',
```
Comment (jirikuncar) [inline .travis.invenio.cfg]: you don't need it

File: .travis.invenio.cfg [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+DEBUG = False
+SECRET_KEY = 'MY_SECRET'
+
+# Disable all automatic asset building - false is /usr/bin/false.
+ASSETS_AUTO_BUILD = False
+
+PACKAGES = [
+    'invenio-oauth2server',
+    'invenio_records',
+    'invenio_testing',
```
Comment (jirikuncar) [inline .travis.invenio.cfg]: remove it

File: .travis.invenio.cfg [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    'invenio_celery',
+    'invenio_ext',
```
Comment (jirikuncar) [inline .travis.invenio.cfg]: remove it

**Comment by jirikuncar** [comment]
- [ ] `include .travis.invenio.cfg` missing in `MANIFEST.in`
- [ ] `invenio_records` import issue

**Comment by SamiHiltunen** [comment]
If you need this for old release, please ping me.

**Pull request closed without merging**